coldfront_overrides/plugin_settings.py

```diff
@@ -1 +1,3 @@
 INSTALLED_APPS += ["imperial_coldfront_plugin"]
+
+from imperial_coldfront_plugin.settings import *
```

Comment:
I believe we don't have to do this if we assign the settings to `Django.conf.settings` in the other app (like was done [here](https://github.com/ImperialCollegeLondon/django-passwordless-login/blob/main/passwordless_login/settings.py))